Ginger/GingerCore/Platforms/Agent.cs

```diff
@@ -692,6 +692,16 @@ public void Close()
                 OnPropertyChanged(Fields.IsWindowExplorerSupportReady);
             }
         }
+
+        public void ResetAgentStatus(eStatus status)
```

Comment:
setting isrunning false is not sufficient. Because agent is still running and we just setting flag to false. Need to close the agent as well